code/ui/manager/src/container/Sidebar.tsx

```diff
@@ -41,9 +42,12 @@ const Sidebar = React.memo(function Sideber({ onMenuClick }: SidebarProps) {
     const whatsNewNotificationsEnabled =
       state.whatsNewData?.status === 'SUCCESS' && !state.disableWhatsNewNotifications;
 
-    const items = api.getElements(types.experimental_SIDEBAR_BOTTOM);
-    const bottom = useMemo(() => Object.values(items), [items]);
-    const top = useMemo(() => Object.values(api.getElements(types.experimental_SIDEBAR_TOP)), []);
+    const bottomItems = api.getElements(Addon_TypesEnum.experimental_SIDEBAR_BOTTOM);
+    const topItems = api.getElements(Addon_TypesEnum.experimental_SIDEBAR_TOP);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    const bottom = useMemo(() => Object.values(bottomItems), [...Object.values(bottomItems)]);
```

Comment:
What is the point of this kind of memoization???

Reply:
You get the same array every time, unless `bottomItems` has new properties/values.

Reply:
Ok, so it's about preserving instance equality of the array so that the consuming UI does not re-render unnecessarily?